### PR TITLE
Update vixen scraper

### PR DESCRIPTION
**Pull request by SpedNSFW**
This commit replaces the old XPath scraper with a new script scraper. This eliminates the need for the CDP.
The script performs faster than the existing scraper (due to no CDP) and even includes scene tags.

Python v3.3 or later is required.

### CODE REVIEW

**Comment by bnkai** [comment]
Can you make the saving json file optional eg an argument like the teamskeet or mg PR ?
While the json file is useful for a lot of reasons some users may simply not want it.
Another cosmetic issue only. Can you sort the url list in the yml file ?
```yml
      - blacked.com/
      - blackedraw.com/
      - deeper.com/
      - tushy.com/
      - tushyraw.com/
      - vixen.com/
  ```
It wasnt enforced till recently but it makes debugging/checking scrapers easier.

File: scrapers/vixenNetwork.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+from urllib.parse import urlparse
+
+
+def log(*s):
+    print(*s, file=sys.stderr)
+
+
+def get_from_url(url_to_parse):
+    p = urlparse(url_to_parse)
+    return p.netloc, p.netloc[4:-4], p.path[1:]
```
Comment (bnkai) [inline scrapers/vixenNetwork.py:15]: This is not actually that safe
In the yaml file we match vixen.com/ so in theory a user could have there `vixen.com/this-whole-time`
Using something like `ltdextract` to get the domain name would be probably better.

Comment (SpedNSFW) [reply scrapers/vixenNetwork.py:15]: Yup! You're right! There are four ways the URL could be inserted and two of them would end up broken. `tldextract` is something that users would have to install so I'm not doing that, especially when there's a simple answer available with RegEx:

```py
m = re.match(r'(?:https?:\/\/)?(?:www\.)?((\w+)\.com)\/([a-z-]+)', url_to_parse)
return  m.groups()
```

File: scrapers/vixenNetwork.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import requests
+import sys
+
+
+def log(*s):
+    print(*s, file=sys.stderr)
+
+
+def get_from_url(url_to_parse):
+    m = re.match(r'(?:https?://)?(?:www\.)?((\w+)\.com)/([a-z-]+)', url_to_parse)
```
Comment (bnkai) [inline scrapers/vixenNetwork.py:13]: `([a-z-]+)`
thats very exclusive
scenes with numbers fail eg `part-1` , `https://www.tushy.com/eva-part-1`

Comment (SpedNSFW) [reply scrapers/vixenNetwork.py:13]: Ahh... That's what you call an oversight.

**Pull request merged**